ModInteract/DeepInteract/FrameBlacklist.java

```diff
@@ -49,25 +50,30 @@ private FramezHandler() {
 
 		@Override
 		@SmartStrip
-		@HandlingPriority(Priority.HIGH)
-		public boolean handleStartMoving(IMovingBlock block) {
+		@Priority(Priority.PriorityEnum.HIGH)
+		public boolean startMoving(IMovingBlock block) {
 			return FrameBlacklist.this.isBlacklisted(block.getWorld(), block.getX(), block.getY(), block.getZ(), block.getBlock(), block.getMetadata(), block.getTileEntity());
 		}
 
 		@Override
 		@SmartStrip
-		@HandlingPriority(Priority.HIGH)
-		public boolean handleFinishMoving(IMovingBlock block) {
+		@Priority(Priority.PriorityEnum.HIGH)
+		public boolean finishMoving(IMovingBlock block) {
 			return FrameBlacklist.this.isBlacklisted(block.getWorld(), block.getX(), block.getY(), block.getZ(), block.getBlock(), block.getMetadata(), block.getTileEntity());
 		}
 
 		@Override
-		@SmartStrip
-		@HandlingPriority(Priority.HIGH)
-		public BlockMovementType getMovementType(World world, Integer x, Integer y, Integer z) {
+		public BlockMovementType getMovementType(World world, int x, int y, int z, ForgeDirection side,
+				IMovement movement) {
 			return FrameBlacklist.this.isBlacklisted(world, x, y, z, world.getBlock(x, y, z), world.getBlockMetadata(x, y, z), world.getTileEntity(x, y, z)) ? BlockMovementType.UNMOVABLE : null;
 		}
 
+		@Override
+		public boolean canHandle(World world, int x, int y, int z) {
+			// TODO Auto-generated method stub
+			return false;
```

Comment:
This should obviously be changed with whatever it needs to be, probably the `isBlacklisted` list check.